packages/quantic/force-app/main/translations/fr.translation-meta.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Translations xmlns="http://soap.sforce.com/2006/04/metadata">
+    <customLabels>
+        <label>Résultat {{0}} de {{1}}</label>
+        <name>quantic_ShowingResultsOf</name>
+    </customLabels>
+    <customLabels>
+        <label>Résultats {{0}} de {{1}}</label>
+        <name>quantic_ShowingResultsOf_plural</name>
+    </customLabels>
+    <customLabels>
+        <label>Résultat {{0}} de {{1}}</label>
```

Comment:
`Aucun résultat`

Reply:
lol yes. I didn't think about it much because it should never show up. you're right.